src/controller/python/BUILD.gn

```diff
@@ -350,6 +350,7 @@ chip_python_wheel_action("chip-clusters") {
         "chip/clusters/TestObjects.py",
         "chip/clusters/Types.py",
         "chip/tlv/__init__.py",
+        "chip/tlv/tlvlist.py",
```

Comment:
tlv/tlvlist.py seems to stutter. could we just name it list ?

Reply:
makes sense